lib/msal-browser/src/config/Configuration.ts

```diff
@@ -323,10 +329,21 @@ export function buildConfiguration(
             appName: Constants.EMPTY_STRING,
             appVersion: Constants.EMPTY_STRING,
         },
+        client: new StubPerformanceClient(
+            DEFAULT_AUTH_OPTIONS.clientId,
+            DEFAULT_AUTH_OPTIONS.authority,
+            new Logger(DEFAULT_LOGGER_OPTIONS, name, version),
+            name,
```

Comment:
Could the PerformanceClient get the name and version itself instead of passing it in the constructor?

Reply:
Unfortunately, we can't do that because 1p and 3p have their own `packageMetadata.ts` files.

Reply:
Could we wrap the client in 1P and set the correct metadata values? Trying to see how we can simplify the constructor, given this will be public

Reply:
Will this also help with setting up the correct SKU if we generalize this?

Reply:
I have simplified `BrowserPerformanceClient` constructor as per your suggestion.